ui-v2/app/templates/dc/acls/policies/index.hbs

```diff
@@ -22,60 +22,63 @@
     {{#block-slot 'content'}}
 {{#if (gt items.length 0) }}
         <form class="filter-bar">
-            {{freetext-filter onchange=(action 'filter') value=filter.s placeholder="Search"}}
+          {{freetext-filter searchable=searchable value=s placeholder="Search"}}
         </form>
 {{/if}}
-{{#if (gt filtered.length 0)}}
-        {{#tabular-collection
-            items=(sort-by 'CreateIndex:desc' 'Name:asc' filtered) as |item index|
-        }}
-            {{#block-slot 'header'}}
-                <th>Name</th>
-                <th>Datacenters</th>
-                <th>Description</th>
-            {{/block-slot}}
-            {{#block-slot 'row' }}
-                <td data-test-policy="{{item.Name}}">
-                  <a href={{href-to 'dc.acls.policies.edit' item.ID}} class={{if (policy/is-management item) 'is-management'}}>{{item.Name}}</a>
-                </td>
-                <td>
-                  {{join ', ' (policy/datacenters item)}}
-                </td>
-                <td>
-                  {{item.Description}}
-                </td>
-            {{/block-slot}}
-            {{#block-slot 'actions' as |index change checked|}}
-                {{#confirmation-dialog confirming=false index=index message="Are you sure you want to delete this Policy?"}}
-                    {{#block-slot 'action' as |confirm|}}
-                        {{#action-group index=index onchange=(action change) checked=(if (eq checked index) 'checked')}}
-                            <ul>
-{{#if (policy/is-management item)}}
-                                <li>
-                                    <a data-test-edit href={{href-to 'dc.acls.policies.edit' item.ID}}>View</a>
-                                </li>
-{{else}}
+        {{#changeable-set dispatcher=searchable}}
+          {{#block-slot 'set' as |filtered|}}
```

Comment:
Here and above you can see the search field component and the changeable-set component 'working together'. Both have access to a `searchable`, the search field (freetext-filter) calls `.search()` on the searchable, and changeable-set listens for change events dispatched by the searchable.

Benefits of this are that you have the searching functionality split off into components, not controllers, but you can still have the search field in one area of the page, and have the results in a completely different area, yet in a 100% ember-like/htmlbars fashion (so as opposed to using something like ember-wormhole).